main.go

```diff
@@ -130,6 +130,8 @@ func cmd() *cobra.Command {
 					set(&context.UpgradeCodeMigration)
 				case "sdk":
 					set(&context.UpgradeSdkVersion)
+				case "pf":
```

Comment:
Should this be included in the `--all` flag?

Reply:
I don't think so because not all providers utilize it?

Reply:
Would this result in confusion if we have a provider that does use it, and we run upgrade --all?I realize this is more about managing expectations.